examples/misc/test/effective_dart_test.dart

```diff
@@ -25,20 +25,29 @@ void main() {
     });
 
     test('runtimeType', () {
-      _test() {
-        // #docregion list-from-2
+      good() {
+        // #docregion list-from-good
         // Creates a List<int>:
         var iterable = [1, 2, 3];
 
         // Prints "List<int>":
         print(iterable.toList().runtimeType);
+        // #enddocregion list-from-good
+      }
+
+      expect(good, prints('List<int>\n'));
+
+      bad() {
```

Comment:
Ditto: bad() -> _bad()

(with the corresponding fix in the expect())

Reply:
Ugh, that lint is dumb. Local functions are never part of APIs.

Changed them to be lambdas.

Reply:
Yeah, I didn't open an issue the last time I faced this, but I have now so that it can get fixed! :) https://github.com/dart-lang/linter/issues/1108